prov/efa/test/efa_test_getinfo.c

```diff
@@ -0,0 +1,97 @@
+#include "efa_unit_tests.h"
+
+int __real_rxr_info_to_rxr(uint32_t version, const struct fi_info *core_info,
+			struct fi_info *info, const struct fi_info *hints);
+
+int __wrap_rxr_info_to_rxr(uint32_t version, const struct fi_info *core_info,
+			struct fi_info *info, const struct fi_info *hints)
+{
+	int use_real = mock_type(bool);
+	int ret;
+	if (use_real)
+		return __real_rxr_info_to_rxr(version, core_info, info, hints);
+
+	ret = mock_type(int);
+	return ret;
+}
+
+/* Currently unused but useful to fake for future unit tests */
+int __real_rxr_get_lower_rdm_info(uint32_t version, const char *node,
+                           const char *service, uint64_t flags,
+                           const struct util_prov *util_prov,
+                           const struct fi_info *util_hints,
+                           struct fi_info **core_info);
+
+int __wrap_rxr_get_lower_rdm_info(uint32_t version, const char *node,
```

Comment:
Instead of mocking `rxr_get_lower_rdm_info`. We need to mock functions like `ibv_open_device`.  `ibv_query_device`, `ibv_query_gid`. Because the behavior is well defined.

Reply:
Sure